tensorflow/python/compiler/tensorrt/trt_convert_test.py

```diff
@@ -702,6 +702,34 @@ def _InputFn():
     del root_with_trt
     gc.collect()  # Force GC to destroy the TRT engine cache.
 
+  @test_util.run_v2_only
+  def testTrtGraphConverter_RemoveNativeSegments(self):
+    """Test case for trt_convert._remove_native_segment()."""
+    np_input = np.random.random_sample([5, 3]).astype(np.float32)
+    # Create a model and save it.
+    input_saved_model_dir = tempfile.mkdtemp(dir=self.get_temp_dir())
+    root = self._GetShapeOpModel()
+    expected_output = root.run(np_input)
+    save.save(root, input_saved_model_dir, signatures=root.run)
+
+    # Run TRT conversion.
+    converter = trt_convert.TrtGraphConverterV2(
+        input_saved_model_dir,
+        precision_mode=trt_convert.TrtPrecisionMode.FP32,
+        allow_build_at_runtime=False,
+        minimum_segment_size=1)
+
+    def _input_fn():
+      yield (np_input,)
+    graph_func = converter.convert()
+    converter.summary()
+    converter.build(_input_fn)
+    # Load and verify the converted model.
+    graph_func = trt_convert._remove_native_segment(graph_func)
```

Comment:
You probably want to run the function to test it. Additionally, it would be good to save & load the model https://github.com/tensorflow/tensorflow/blob/4f8b87739e3f3c4aee0e36714e16856227c83e9a/tensorflow/python/compiler/tensorrt/trt_convert_test.py#L1027-L1032
and infer the loaded model https://github.com/tensorflow/tensorflow/blob/4f8b87739e3f3c4aee0e36714e16856227c83e9a/tensorflow/python/compiler/tensorrt/trt_convert_test.py#L1052